lib/logstash/plugin_mixins/aws_config/v1.rb

```diff
@@ -0,0 +1,84 @@
+module LogStash::PluginMixins::AwsConfig::V1
+  # This method is called when someone includes this module
```

Comment:
This is comment necessary? (I know you just copies here as it was before) (_small style comment_)